python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -207,12 +346,139 @@ def parse_stream_tables(
         return tables
 
 
+class AirbyteInstanceCacheableAssetsDefintion(CacheableAssetsDefinition):
+    def __init__(
+        self,
+        airbyte_resource_def: ResourceDefinition,
+        workspace_id: Optional[str],
+        key_prefix: List[str],
+        create_assets_for_normalization_tables: bool,
+        connection_to_group_fn: Optional[Callable[[str], Optional[str]]],
+    ):
+        self._airbyte_resource_def = airbyte_resource_def
+        self._airbyte_instance: AirbyteResource = airbyte_resource_def(
+            build_init_resource_context()
+        )
+
+        self._workspace_id = workspace_id
+        self._key_prefix = key_prefix
+        self._create_assets_for_normalization_tables = create_assets_for_normalization_tables
+        self._connection_to_group_fn = connection_to_group_fn
+
+        super().__init__(unique_id="airbyte")
```

Comment:
this unique_id should probably be a hash of some of these input properties (It's definitely pretty unlikely to want to get assets from multiple airbyte instances in a single repository, but best to just avoid collisions in general)